README.md

```diff
@@ -243,6 +283,10 @@ Known issues:
 
 ### How to upgrade from v6.x.x to v7.x.x?
 
+- Themes have been introduced and should be set as needed to retain the correct look and feel, see [theme documentation](#themes).
+- The following colour usecases are renamed:
```

Comment:
(Again, remove passive voice - http://www.hemingwayapp.com/ is useful for autochecking this if you find it difficult to spot)
"We have renamed the following usecases"